eng/create-tag.ps1

```diff
@@ -1,19 +1,27 @@
-params(
+param(
 	$AccessToken
 )
 
 $Commit = $env:Build.SourceVersion
 $Version = $env:VersionPrefix
 
-$uri ="$($env:System_TeamFoundationCollectionUri)/$($env:System_TeamProject)/_apis/git/repositories/$($env:Build_Repository_Name)/annotatedTags?api-version=4.1-preview.1"
+$uri ="$($env:System_TeamFoundationCollectionUri)/$($env:System_TeamProject)/_apis/git/repositories/$($env:Build_Repository_Name)/annotatedTags?api-version=5.0-preview.1"
 $tag = "v$Version"
 
 if ($Commit -and $Version) {
 	Write-Output "Tagging $Commit with $tag"
```

Comment:
NIT: it looks like there is some mixed tabs and spaces here. Please fix.

Reply:
FIxed